aws/templates/id/id_apigateway.ftl

```diff
@@ -1,32 +1,33 @@
 [#-- API Gateway --]
 
-[#assign APIGATEWAY_COMPONENT_TYPE = "apigateway"]
-
-[#assign APIGATEWAY_RESOURCE_TYPE = "apigateway"]
-[#assign APIGATEWAY_DEPLOY_RESOURCE_TYPE = "apiDeploy"]
-[#assign APIGATEWAY_STAGE_RESOURCE_TYPE = "apiStage"]
-[#assign APIGATEWAY_DOMAIN_RESOURCE_TYPE = "apiDomain"]
-[#assign APIGATEWAY_AUTHORIZER_RESOURCE_TYPE = "apiAuthorizer"]
-[#assign APIGATEWAY_BASEPATHMAPPING_RESOURCE_TYPE = "apiBasePathMapping"]
-[#assign APIGATEWAY_USAGEPLAN_RESOURCE_TYPE = "apiUsagePlan"]
-[#assign APIGATEWAY_APIKEY_RESOURCE_TYPE = "apiKey"]
-
-[#assign APIGATEWAY_DOCS_EXTENSION = "docs"]
+[#-- Resources --]
+[#assign AWS_APIGATEWAY_RESOURCE_TYPE = "apigateway"]
+[#assign AWS_APIGATEWAY_DEPLOY_RESOURCE_TYPE = "apiDeploy"]
+[#assign AWS_APIGATEWAY_STAGE_RESOURCE_TYPE = "apiStage"]
+[#assign AWS_APIGATEWAY_DOMAIN_RESOURCE_TYPE = "apiDomain"]
+[#assign AWS_APIGATEWAY_AUTHORIZER_RESOURCE_TYPE = "apiAuthorizer"]
+[#assign AWS_APIGATEWAY_BASEPATHMAPPING_RESOURCE_TYPE = "apiBasePathMapping"]
+[#assign AWS_APIGATEWAY_USAGEPLAN_RESOURCE_TYPE = "apiUsagePlan"]
+[#assign AWS_APIGATEWAY_APIKEY_RESOURCE_TYPE = "apiKey"]
 
 [#function formatDependentAPIGatewayAuthorizerId resourceId extensions...]
     [#return formatDependentResourceId(
-                APIGATEWAY_AUTHORIZER_RESOURCE_TYPE,
+                AWS_APIGATEWAY_AUTHORIZER_RESOURCE_TYPE,
                 resourceId,
                 extensions)]
 [/#function]
 
 [#function formatDependentAPIGatewayAPIKeyId resourceId extensions...]
     [#return formatDependentResourceId(
-                APIGATEWAY_APIKEY_RESOURCE_TYPE,
+                AWS_APIGATEWAY_APIKEY_RESOURCE_TYPE,
                 resourceId,
                 extensions)]
 [/#function]
 
+[#-- Component --]
+[#assign APIGATEWAY_COMPONENT_TYPE = "apigateway"]
+[#assign APIGATEWAY_COMPONENT_DOCS_EXTENSION = "docs"]
```

Comment:
Think this is a resource extension

Reply:
The Docs are part of the component deployment though. They aren't part of API Gateway itself. The S3 bucket is the resource and in the API Gateway component it gets a docs extension.